ngi_reports/common/ign_sample_report.py

```diff
@@ -62,11 +60,6 @@ def __init__(self, config, LOG, working_dir, **kwargs):
         self.parse_picard_metrics()
         self.parse_deliveries()
 
-        self.create_aggregate_statistics()
-
-        self.LOG.debug('Plotting graphs')
-        self.make_plots()
```

Comment:
Yep, I'll add a comment to my code that should tag along with future PRs. Having the `ign_aggregate_report.CommonReport` subclass the `ing_sample_report.CommonReport` is probably a very good way of ensuring confusion and annoyance, but right now, I'm just lazy and I think I'll leave it like that if you don't mind.

I would suggest to actually keep the function call in the node-specific classes. That way, there is the possibility to make node-specific alterations to the data structures, if needed, both before the call and after the call but before sending control back to the common script, which I think is quite nice.